include/libtorrent/piece_block.hpp

```diff
@@ -48,7 +48,7 @@ namespace libtorrent
 		{
 		}
 		piece_index_t piece_index;
-		int block_index;
+		int block_index = 0;
```

Comment:
piece_index_t piece_index = {0};
VC++2015 error:
1>q:\vc15\r5xx\libtorrent\include\libtorrent\piece_block.hpp(50): error C2664: 'libtorrent::aux::strong_typedef<int32_t,libtorrent::aux::piece_index_tag,void>::strong_typedef(const libtorrent::aux::strong_typedef<int32_t,libtorrent::aux::piece_index_tag,void> &)': cannot convert argument 1 from 'int' to 'const libtorrent::aux::strong_typedef<int32_t,libtorrent::aux::piece_index_tag,void> &'
1>  q:\vc15\r5xx\libtorrent\include\libtorrent\piece_block.hpp(50): note: Reason: cannot convert from 'int' to 'const libtorrent::aux::strong_typedef<int32_t,libtorrent::aux::piece_index_tag,void>'
1>  q:\vc15\r5xx\libtorrent\include\libtorrent\piece_block.hpp(50): note: Constructor for struct 'libtorrent::aux::strong_typedef<int32_t,libtorrent::aux::piece_index_tag,void>' is declared 'explicit'

help?
or revert int block_index = 0 ?